microsite/blog/2023-04-26-kubecon-eu-2023.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'ICYMI: KubeCon EU 2023'
+author: Bailey Brooks, Spotify
+authorURL: http://github.com/bailey
+authorImageURL: https://pbs.twimg.com/profile_images/1477424303192694785/qCfN6XWW_400x400.jpg
+---
+
+![KubeCon CloudNativeCon Europe 2023 banner](assets/2023-04-26/Backstage_News_3-7-23_KubeCon.png)
+
+We had a blast talking all things Backstage at KubeCon EU 2023 in Amsterdam last week. In case you missed it, here’s a quick recap:
+
```

Comment:
You'll want to add a truncate comment here for the fold, see other blog posts. Cba looking up the exact format as I'm on mobile :sweat_smile: